setup.py

```diff
@@ -12,6 +12,7 @@
 os.system('mkdir -p ~/.local/share/koneko')
 os.system('cp -r ./pics/ ~/.local/share/koneko/')
 os.system('cp example_config.ini ~/.local/share/koneko/')
+os.system('cp pixiv-url.desktop ~/.local/share/koneko/')
```

Comment:
Don't you mean `~/.local/share/applications/` here?

Reply:
I don't want to install it directly there. Say a user who already has a config+token reinstalls `koneko`, we wouldn't want them to end up with a useless `.desktop` file potentially conflicting with another app. I think `koneko` should copy the file to `~/.local/share/applications` when it is run.